zone/client.h

```diff
@@ -1853,6 +1882,14 @@ class Client : public Mob
 	bool Trader;
 	bool Buyer;
 	std::string BuyerWelcomeMessage;
+	int32                                            parcel_platinum;
+	int32                                            parcel_gold;
+	int32                                            parcel_silver;
+	int32                                            parcel_copper;
+	int32                                            parcel_count;
+	bool                                             parcel_enabled;
+	bool                                             parcel_merchant_engaged;
+	std::map<uint32, BaseParcelsRepository::Parcels> parcels{};
```

Comment:
Any private member variables we've been trying to prefix with `m_` when we can

eg `m_parcel_platinum`

Clion can easily refactor these accurately

Reply:
Resolved.  Mental note.  Clion auto refactor - Deselect within text areas.  :-)